open/text/embeddings/server/app.py

```diff
@@ -62,39 +63,48 @@ class CreateEmbeddingResponse(BaseModel):
 
 embeddings = None
 
-
-def _create_embedding(
-    model: Optional[str],
-    input: Union[str, List[str]]
-):
+def initialize_embeddings(model: Optional[str] = None):
```

Comment:
Thanks for send me the PR. 

It make sense to load the text embeddings model before the first request. I leave the code of loading the embedding model on-demand back then due to I haven't make up my mind should the API serve multiple models or single model. I think it is clear now that the API serve single text embeddings model is the way to go, the code is simpler. If another text embeddings model is needed, the user can simply deploy another instance of the API. 

Hence, the `model` parameter no longer needed here:
```suggestion
def initialize_embeddings():
```